### PR TITLE
Chore/Dao Selector - DAO-704

### PR DESCRIPTION
**Pull request by Fabricevladimir**
- refactors web-app dao switcher for mobile and desktop
- exports and uses dropdown for breadcrumb dropdown
- removes initial breadcrumb menu
- refactors NavLinks component

### CODE REVIEW

**Review by RakeshUP — APPROVED** [review]
Nice. Looks great

File: packages/ui-components/src/components/dropdown/dropdown.tsx [hunk 1]
```diff
@@ -41,21 +41,18 @@ export type DropdownProps = CustomDropdownContentProps & {
...
       <DropdownMenu.Trigger data-testid="dropdown-trigger">
         {trigger}
       </DropdownMenu.Trigger>
 
-      <StyledContent>
+      <StyledContent {...rest}>
         <DropdownMenu.Group className="space-y-0.5">
-          {listItems?.map(li => (
-            <StyledItem
-              key={li.component?.toString()}
```
Comment (WissenIstNacht) [inline packages/ui-components/src/components/dropdown/dropdown.tsx:55]: yeah, that didn't quite work out as planned :joy:

**Review by WissenIstNacht — APPROVED** [review]
Thanks a lot, looks great!

Are we done with Navbars for a while now? haha

File: packages/ui-components/src/components/dropdown/dropdown.tsx [hunk 2]
```diff
@@ -41,21 +41,18 @@ export type DropdownProps = CustomDropdownContentProps & {
...
-              onSelect={li.callback}
-              style={{}}
-            >
+          {listItems?.map((li, index) => (
+            <StyledItem key={index} onSelect={li.callback} style={{}}>
```
Comment (sepehr2github) [inline packages/ui-components/src/components/dropdown/dropdown.tsx:55]: Why did an empty style pass to this component?

**Review by sepehr2github — APPROVED** [review]
Finally! hahaha. Thanks Fabrice!

File: packages/ui-components/src/components/dropdown/dropdown.tsx (hunk 2 above)
Comment (Fabricevladimir) [reply packages/ui-components/src/components/dropdown/dropdown.tsx:55]: Forgot to remove it 🙈

File: packages/ui-components/src/components/dropdown/dropdown.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/ui-components/src/components/dropdown/dropdown.tsx:55]: It's the thought that counts lol

**Comment by Fabricevladimir** [comment]
> Thanks a lot, looks great!
> 
> Are we done with Navbars for a while now? haha

No 🥲

**Pull request merged**